src/storm/values.yaml

```diff
@@ -78,3 +81,5 @@ zookeeper:
   #  - "example.external.zookeeper3"
   # # external server port
   # port: 2181
+worker:
+  childopts: "-Xmx%HEAP-MEM%m -XX:+PrintGCDetails -Xloggc:artifacts/gc.log -XX:+PrintGCDateStamps -XX:+PrintGCTimeStamps -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=10 -XX:GCLogFileSize=1M -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=artifacts/heapdump -ea"
```

Comment:
any `worker` options belong to `supervisor`, as it's the one responsible for spawning workers

Reply:
should I remove `worker.childopts` in that case? 🤔

Reply:
I think we could remove it for now, yes.

To go back to my initial comment, please be sure to rename all the `worker.*` values to `supervisor.worker.*` in `values.yaml`!

Reply:
done 🚀